test/integration/golden_response/staging/get_stat_collection/US_State.json

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
@chejennifer @pradh @beets PTAL the new response format

Reply:
This response format looks good to me

Reply:
should we include a pointer to the import per stat? so we'd know which source to attribute to for each datapoint? the use case would be for the tooltip to be specific in its attribution. one could argue that this works for the entire collection, and the other would be the result of /stat/value anyway. thoughts?

Reply:
Added the import name in each stat value. That can reference the full source information in the response by import name.